js/pulse.js

```diff
@@ -587,6 +689,39 @@
                 return false;
             };
 
+            this.showOverlayAd = function (pulseOverlayAd) {
```

Comment:
Who calls this function? I don't see showOverlayAd called in this file. Is the Pulse SDK going to call this?